Alueta_MarkRoland/Activity3/Resume/resume.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Activity 3 - Online Resume of Mark Roland C. Alueta - Advance</title>
+    <link rel="stylesheet" href="index.css" />
+  </head>
+  <body>
+    <center>
```

Comment:
Since you are already advance, using center tag is deprecated. you can try to use
display:flex; justify-content:center; text-align:center